lua/rustaceanvim/shell.lua

```diff
@@ -1,11 +1,16 @@
 local M = {}
 
 ---@return boolean
-local function is_windows()
+function M.is_windows()
```

Comment:
issue: calls of `is_windows` in this file have to be replaced with `M.is_windows`